clients/android/app/src/main/java/app/lockbook/ui/HandwritingEditorView.kt

```diff
@@ -286,6 +344,7 @@ class HandwritingEditorView(context: Context, attributeSet: AttributeSet?) :
 
     private fun moveTo(point: PointF, pressure: Float) {
         lastPoint.set(point)
+        strokesBounds.add(RectF(point.x, point.y, point.x, point.y))
```

Comment:
Doesn't this mean that the size of `strokeBounds` will be roughly equal to the size of `points`. Don't we want the size of `strokeBounds` to be equal to the number of strokes?